database/benchmarkMongo.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+const Promise = require('bluebird');
+const fs = require('fs');
+const helper = require('./index');
+
+const getRandom = (min, max) => {
+  const a = Math.ceil(min);
+  const b = Math.floor(max);
+  return Math.floor(Math.random() * ((b - a) + 1)) + a;
+}; 
+
+let timeArray = [];
```

Comment:
I am wondering if you could save yourself computation power by just adding each time instead of storing into an array then you could divide by 10000 since that is the length of the array anyways. Just an idea